src/CoreBundle/Exception/MemcachedAccessFailedException.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace ChameleonSystem\CoreBundle\Exception;
+
+class MemcachedAccessFailedException extends \Exception
```

Comment:
Just one remark by me: I'd prefer to have a SessionAccessFailedException or similar - the fact that the session uses memcached shouldn't bleed into client code as this is an implementation detail that may change anytime.

Reply:
done